.github/workflows/files-changed.yml

```diff
@@ -35,6 +35,8 @@ jobs:
       - uses: dorny/paths-filter@v2
         id: changes
         with:
+          # used for `act_runner exec`
+          # base: HEAD
```

Comment:
```suggestion
```

Outdated comment?

Reply:
not outdated, means should add this line when using `act_runner exec`, else can't run `act_runner exec`. (follow https://github.com/go-gitea/gitea/pull/25524#issuecomment-1613505758)

Reply:
Hmm, can we always add it?

Reply:
looks not, they are diferent compare mode.